src/app/screens/Onboard/TestConnection/index.tsx

```diff
@@ -17,7 +17,7 @@ export default function TestConnection() {
 
   const navigate = useNavigate();
 
-  async function handleEdit(event: React.MouseEvent<HTMLButtonElement>) {
+  async function handleEdit() {
```

Comment:
Are there any changes required in this component? I see there are two different components for testing connection (onboarding vs adding an account from the options).

Please note: I do not fully understand the code in TestConnection/AccountContext. Is there someone who has more knowledge of the code who might be able to review this? @escapedcat

Reply:
This looks like a "cleanup only", right @lujakob ? It's not needed but can be done I guess. Could be done in a different PR top keep things scoped but not a blocker really.
@rolznz is right, we have two of these. One for "adding an accoiunt via welcome/onboarding" and another one for "adding a new account after the onboarding already happened an d an accoiunt exists", which is here:
https://github.com/getAlby/lightning-browser-extension/blob/master/src/app/screens/Options/TestConnection/index.tsx#L34-L37

Does this help?

Reply:
Yes that's a minor cleanup. Should I revert it, to be addressed at another point?

There are two TestConnection components. The "loadAccountInfo" handler is different however. I guess they are different.